### PR TITLE
Update/identity list subscriptions

### PR DESCRIPTION
**Pull request by lleonard-msft**
Initial draft of `/subscription` changes for review.

### CODE REVIEW

File: api-reference/beta/api/subscription_list.md [hunk 1]
```diff
@@ -0,0 +1,84 @@
+# List subscriptions
+
+> **Important:** APIs under the /beta version in Microsoft Graph are in preview and are subject to change. Use of these APIs in production applications is not supported.
+
+Retrieve the properties and relationships of a subscription.
```
Comment (dkershaw10) [inline api-reference/beta/api/subscription_list.md]: I hate this boilerplate stuff. It doesn't really tell me anything useful.  You may as well remove the sentence for all its value.  Also if I recollect from the API review, there were some subtleties to what is actually returned.

GET API doesn't return All subscriptions, but only the subscriptions for the signed-in user and the calling application combo if the GET call is made in the Delegated context. Same GET API will return all subscriptions created with the Application if the GET call is made in App Only context.

File: api-reference/beta/api/subscription_list.md [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+| Users                       | User.Read.All       |
+| Drive  (User's OneDrive)    | Files.ReadWrite     |
+| Drives (Sharepoint shared content and drives) | Files.ReadWrite.All |
+
+***Note:*** The /beta endpoint allows application permissions for most resources. Conversations in a Group and OneDrive drive root items are not supported with application permissions.
+
+## HTTP request
+<!-- { "blockType": "ignored" } -->
+```http
+GET /subscriptions/
```
Comment (dkershaw10) [inline api-reference/beta/api/subscription_list.md]: Remove the trailing /

File: api-reference/beta/api/subscription_list.md [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+
+## Request body
+Do not supply a request body for this method.
+
+## Response
+
+If successful, this method returns a `200 OK` response code and [subscription](../resources/subscription.md) object in the response body.
+## Example
+##### Request
+Here is an example of the request.
```
Comment (dkershaw10) [inline api-reference/beta/api/subscription_list.md]: Remove this sentence.  Adds no value.  More boiler plate stuff

File: api-reference/beta/api/subscription_list.md [hunk 4]
```diff
@@ -0,0 +1,84 @@
...
+<!-- {
+  "blockType": "request",
+  "name": "get_subscription"
+}-->
+```http
+GET https://graph.microsoft.com/beta/subscriptions/{id}
+```
+##### Response
+Here is an example of the response.
```
Comment (dkershaw10) [inline api-reference/beta/api/subscription_list.md]: Remove this sentence.  Adds no value.  More boiler plate stuff

File: api-reference/beta/api/subscription_list.md [hunk 5]
```diff
@@ -0,0 +1,84 @@
...
+If successful, this method returns a `200 OK` response code and [subscription](../resources/subscription.md) object in the response body.
+## Example
+##### Request
+Here is an example of the request.
+<!-- {
+  "blockType": "request",
+  "name": "get_subscription"
+}-->
+```http
+GET https://graph.microsoft.com/beta/subscriptions/{id}
```
Comment (dkershaw10) [inline api-reference/beta/api/subscription_list.md]: This is a GET single object request.  However this is the list topic.  Remove the trailing /{id}

File: api-reference/beta/api/subscription_list.md [hunk 6]
```diff
@@ -0,0 +1,84 @@
...
+  "blockType": "response",
+  "truncated": false,
+  "@odata.type": "microsoft.graph.subscription"
+} -->
+```http
+HTTP/1.1 200 OK
+Content-type: application/json
+Content-length: 252
+
+{
```
Comment (dkershaw10) [inline api-reference/beta/api/subscription_list.md]: This response is not a collection, which is what you'd expect from a list operation

File: api-reference/beta/api/subscription_list.md [hunk 7]
```diff
@@ -0,0 +1,84 @@
...
+  "expirationDateTime":"2016-11-20T18:23:45.9356913Z",
+  "creatorUserId": "string"
+}
+```
+
+<!-- uuid: 8fcb5dbc-d5aa-4681-8e31-b001d5168d79
+2015-10-25 14:57:30 UTC -->
+<!-- {
+  "type": "#page.annotation",
+  "description": "Get subscription",
```
Comment (dkershaw10) [inline api-reference/beta/api/subscription_list.md]: Change to list.  I guess you copied this from the GET topic ;)

File: api-reference/beta/api/subscription_list.md [hunk 8]
```diff
@@ -0,0 +1,84 @@
...
+```http
+GET https://graph.microsoft.com/beta/subscriptions/{id}
+```
+##### Response
+Here is an example of the response.
+<!-- {
+  "blockType": "response",
+  "truncated": false,
+  "@odata.type": "microsoft.graph.subscription"
+} -->
```
Comment (dkershaw10) [inline api-reference/beta/api/subscription_list.md]: I believe this should contain something like "collection": true.  Please check another topic that has a collection response.

File: api-reference/beta/resources/subscription.md [hunk 9]
```diff
@@ -26,9 +26,11 @@ Here is a JSON representation of the resource.
   "changeType": "string",
   "notificationUrl": "string",
   "resource": "string",
+  "applicationId" : "string",
```
Comment (dkershaw10) [inline api-reference/beta/resources/subscription.md]: The intro doesn't talk about support for Users and Groups.  Also, notifications are about **changes** to data and not just about data.  @jesakowi Can you provide a better intro?

File: api-reference/beta/resources/subscription.md [hunk 10]
```diff
@@ -63,6 +68,7 @@ None
 |:-----------------------------------------------------------------|:--------------------------------|:----------------------------------------------------------------------------------------------|
 | [Create subscription](../api/subscription_post_subscriptions.md) | [subscription](subscription.md) | Subscribes a listener application to receive notifications when Microsoft Graph data changes. |
 | [Update subscription](../api/subscription_update.md)             | [subscription](subscription.md) | Renew a subscription by updating its expiration time.                                         |
+| [List subscriptions](../api/subscription_list.md)                | [subscription](subscription.md) | Lists activie subscriptions. |
```
Comment (dkershaw10) [inline api-reference/beta/resources/subscription.md]: Typo.  "activie"?

**Comment by jesakowi** [comment]
Reviewed- looks good to me

**Pull request merged**